apps/src/code-studio/components/stageExtras/StageExtras.story.jsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 import StageExtras from './StageExtras';
+import _ from 'lodash';
+
+function generateFakePersonalProjects() {
+  return _.range(5).map(i => (
+    {
+      name: "Personal " + i,
+      updatedAt: Date.now() - i * 60 * 1000,
+      projectType: 'gamelab',
+      id: 'abcd'
+    }
+  ));
+}
```

Comment:
[Duplicated code from ProjectWidget.story?](https://github.com/code-dot-org/code-dot-org/blob/1c0f1d93889f65f53b690ea3cfc3f192179c555a/apps/src/templates/projects/ProjectWidget.story.jsx#L5-L17)  Would be good to extract both of these to a shared helper.